examples/Dockerfile.service

```diff
@@ -0,0 +1,64 @@
+# Define the Container registry for our base images, as these are not stored on Dockerhub.
+#   You will need to be logged into the registry in order to build this.
+ARG REGISTRY=562955126301.dkr.ecr.eu-west-2.amazonaws.com
+ARG BASE_VERSION=1.0.0
+ARG DEV_VERSION=1.0.0
+FROM $REGISTRY/ffc-node-base:$BASE_VERSION
+
+# Make sure that we don't run as root. The "node" user is set up in the base node version that our base parent image uses.
+USER node
+
+WORKDIR /home/node
+
+# Copy our package files so that our package install will do a clean install. This installs the exact versions of the packages
+#   listed in package-lock.json, and does not update either the package-lock.json or the package.json file.
+# Our production dependancies are now installed.
+COPY --chown=node:node package*.json ./
+RUN npm ci
+
+# Development stage installs devDependencies, builds app from source and declares a file watcher as the default command.
+#   We name this stage so we can refer to it in later stages
+FROM $REGISTRY/ffc-node-development:$DEV_VERSION AS development
+
+# Expose some development debugging ports that are used during development
+EXPOSE 9229 9230
+
+# As this is a new container, we want to ensure that we set up the working directory and user correctly
+USER node
+WORKDIR /home/node
+
+# We copy the installed packages from the production "base" install that was the first stage.
+COPY --from=base --chown=node:node /home/node/package*.json ./
+COPY --from=base --chown=node:node /home/node/node_modules ./node_modules

```

Comment:
There doesn't seem to be much benefit in copying the node modules and files installed during the base image build into the development build.

A straight copy of the package files from the local folder followed by the npm install reduces the build time from 63 seconds to 48 on my system. It's slower copying and merging two installs that a straight install.

Reply:
I've changed it to remove the copy line and just rely on the npm install that occurs in the next step